modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/nowplaying/PlayerHeadingSection.kt

```diff
@@ -0,0 +1,327 @@
+                        alpha = if (state.isFirstChapter) 0.5f else 1f,
+                    )
+                }
+
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    modifier = Modifier
+                        .padding(top = 8.dp)
+                        .padding(horizontal = 16.dp)
+                        .weight(1f),
+                ) {
+                    TextH30(
+                        text = state.title,
+                        color = Color(ThemeColor.playerContrast01(state.theme)),
+                        textAlign = TextAlign.Center,
+                        maxLines = 2,
+                        modifier = Modifier
+                            .then(if (state.isChaptersPresent) Modifier.clickable { state.chapter?.let { onChapterTitleClick(it) } } else Modifier),
+                    )
+
+                    if (!state.isChaptersPresent) {
+                        state.podcastTitle?.takeIf { it.isNotBlank() }?.let {
+                            Spacer(
+                                modifier = Modifier.height(4.dp),
+                            )
+
+                            TextH50(
+                                text = state.podcastTitle,
+                                color = MaterialTheme.theme.colors.playerContrast02,
+                                maxLines = 1,
+                                modifier = Modifier
+                                    .clickable { onPodcastTitleClick() },
+                            )
+                        }
+                    }
+
+                    if (state.isChaptersPresent) {
+                        state.chapterSummary.takeIf { it.isNotBlank() }?.let {
+                            Spacer(
+                                modifier = Modifier.height(4.dp),
+                            )
+
+                            TextH70(
+                                text = it,
+                                maxLines = 1,
+                                color = Color.White.copy(alpha = 0.7f),
+                            )
+                        }
+                    }
+                }
+
+                if (state.isChaptersPresent) {
+                    Column(
+                        horizontalAlignment = Alignment.CenterHorizontally,
+                    ) {
+                        ChapterNextButtonWithChapterProgressCircle(
+                            onClick = onNextChapterClick,
+                            enabled = !state.isLastChapter,
+                            alpha = if (state.isLastChapter) 0.5f else 1f,
+                        )
+
+                        Spacer(modifier = Modifier.height(4.dp))
+
+                        TextH70(
+                            text = state.chapterTimeRemaining,
+                            color = Color.White,
+                            modifier = Modifier
+                                .alpha(0.4f),
```

Comment:
<img  width="300" src="https://github.com/user-attachments/assets/f4571cc8-4413-489e-b39b-d66f67c8ef2f" /> 

I wonder if it would be good to say "35s left in the chapter"

<img  width="300" src="https://github.com/user-attachments/assets/7fbd9a22-713d-49ff-975e-f458aac533b6" /> 

And "1 of 9 chapters"

Reply:
Good idea! Updated in 9a7ee6e and 798961c.